src/Mod/TechDraw/Gui/QGISVGTemplate.cpp

```diff
@@ -250,45 +268,35 @@ void QGISVGTemplate::createClickHandles()
     });
 }
 
-//! find the font-size hidden in a style element
-double QGISVGTemplate::getFontSizeFromStyle(QString style)
-{
-    if (style.isEmpty()) {
-        return 0.0;
-    }
 
-    // get text attributes from style element
-    QRegularExpression rxFontSize(QStringLiteral("font-size:([0-9]*.?[0-9]*)px;"));
-    QRegularExpressionMatch match;
-
-    int pos{0};
-    pos = style.indexOf(rxFontSize, 0, &match);
-    if (pos != -1) {
-        return Rez::guiX(match.captured(1).toDouble());
-    }
+//! estimate the size of the required clickbox from the font family and font size
+QSizeF QGISVGTemplate::calculateClickboxSize(const QString& editableValue,
+                                             const TechDraw::SvgTextAttributes& attributes) const
+{
+    constexpr double PixelsPerMM{3.78};     // based on CSS 96px / inch
+    constexpr double StdCSSDpi{96};
```

Comment:
Can we really hard code this?
I think my monitors dpi is around 200-250dpi (mac default)

If it is correct, how about this instead?
```suggestion
    constexpr double StdCSSDpi{96};
    constexpr double PixelsPerMM{StdCSSDpi / 25.4};
```

Reply:
It has nothing to do with the monitor (I think).  This is used for scaling QFontMetricsF pixel numbers to make them the same size (ish) as what the SVG renderer produces.